SwiftUiMovieDb/Views/MovieBackdropCard.swift

```diff
@@ -11,27 +11,37 @@ import Kingfisher
 struct MovieBackdropCard: View {
     
     let movie: Movie
+    private let aspectRatio: CGFloat = 16/9
+    private let cornerRadius: CGFloat = 8
+    private let shadowRadius: CGFloat = 4
+    private let lineLimit = 1
     
     var body: some View {
         VStack(alignment: .leading) {
             ZStack {
                 Rectangle()
                     .fill(Color.gray.opacity(0.3))
                 
-                KFImage(movie.backdropURL)                    
+                KFImage(movie.backdropURL)
+                    .placeholder(backdropPlaceholderImage)
                     .cacheMemoryOnly()
-                    .resizable()
-                    .aspectRatio(contentMode: /*@START_MENU_TOKEN@*/.fill/*@END_MENU_TOKEN@*/)                   
+                    .resizable()                  
             }
-            .aspectRatio(16/9, contentMode: .fit)
-            .cornerRadius(8)
-            .shadow(radius: 4)
+            .aspectRatio(aspectRatio, contentMode: .fit)
+            .cornerRadius(cornerRadius)
+            .shadow(radius: shadowRadius)
             Text(movie.title)
         }
-        .lineLimit(1)
+        .lineLimit(lineLimit)
     }
 }
 
+@ViewBuilder
+func backdropPlaceholderImage() -> some View {
+    Image("PosterImagePlaceholder")
```

Comment:
Let's create an enum to manage all the icon or image asset to be typesafe and less errorprone, avoid using String constant's.